### ISSUE TITLE
snapshot migrate error

### ISSUE THREAD

**Issue by cristihcd** [kopia/kopia#3863]
Using kopia 0.17, on Windows X64, with a NAS server repository (SMB share). 

When running: `kopia --config-file repository-remote.config snapshot migrate --all --no-policies --source-config=c:\Users\User\AppData\Roaming\kopia\repository.config`, I get the following error: `ERROR unable to migrate source: error migrating snapshot [redacted] @ 1701086834924508500: not a local filesystem`. 

Full log attached.

OS snapshots (VSS) are enabled for source and destination repo:

```json
  "osSnapshots": {
    "volumeShadowCopy": {
      "enable": 2
    }
  },
```
[migrate.log](https://github.com/kopia/kopia/files/15296602/migrate.log)

**Comment by cristihcd** [comment]
After disabling VSS snapshot on the destination repo, the migration runs successfully

**Comment by jkowalski** [comment]
yeah, but this is undesireable. we should ignore OS Snapshots in migration mode.
